src/new-client/src/app/site/[site]/view-availability/page.tsx

```diff
@@ -0,0 +1,59 @@
+import NhsPage from '@components/nhs-page';
+import { fetchAvailability, fetchSite } from '@services/appointmentsService';
+import dayjs from 'dayjs';
+import { ViewAvailabilityPage } from './view-availability-page';
+import { FetchAvailabilityRequest } from '@types';
+import {
+  getDetailedMonthView,
+  getWeeksInMonth,
+} from '@services/viewAvailabilityService';
+
+type PageProps = {
+  params: {
+    site: string;
+  };
+};
+
+const Page = async ({ params }: PageProps) => {
```

Comment:
Could we add a `await assertPermission(site.id, 'availability:query')` to this so the route is protected with the same permissions that govern the card's visibility?